[Pategara] Created study overview and individual topic overview

ChrisPy-Pata opened the pull request:
**Notes:** 
- Created temporary lists for goal and topic information. 

[Not Yet Working/Functional]:
- Search Bar
- Sort Function
- Buttons ( Start Session, Confirm)                                                                                                                                                                      - Having problems with preventing touch/press exit when popup card is present. ( like if i press ni user ang outside sa popupcard kay ga exit siya :>. Can't find solution yet.

breamier reviewed (changes requested):
Please resolve the following before merging:

- Overflow

![489041657_578710371912529_4562500060511721987_n](https://github.com/user-attachments/assets/07e67328-5092-4518-84b2-082bc2dc38ad)

![488589675_1662883744615119_7504103054819248893_n](https://github.com/user-attachments/assets/88271627-d637-43a9-b2ea-50b4ff84f2ad)

- Change font to white when searching
![490236572_1003614074680616_5405688128943491656_n](https://github.com/user-attachments/assets/fefddc26-808c-4745-a950-efc19870640c)

ChrisPy-Pata commented:
Fixed Problems: Overflow and added textstyle on searchbar text input
Changes: made dimension screen responsive

ChrisPy-Pata commented:
[Changes Made]

1. Fixed Button Overflow 
2. Implement MediaQuery to most width and height of widgets
3. Merge changes from tutorial_screen


[Notes]

1. Start Session and Confirm button [ not yet implemented]
2. Delete icon will pop topic_overview_screen page, since its onpressed function can't be not [TO BE CHANGE]
3. Photo gallery [ not yet implemented]
4. Searchbar and Sorting [Not yet implemented]
5. added temporary data to display plan design [ TO BE CHANGE]
